resources/ctia-default.properties

```diff
@@ -68,9 +68,9 @@ ctia.store.es.default.port=9205
 #ctia.store.es.default.host=es-storage.int.iroh.site
 #ctia.store.es.default.port=443
 #ctia.store.es.default.protocol=https
-#ctia.store.es.default.auth.type=basic-auth
-#ctia.store.es.default.auth.params.user=elastic
-#ctia.store.es.default.auth.params.pwd=ductile
+ctia.store.es.default.auth.type=basic-auth
+ctia.store.es.default.auth.params.user=elastic
+ctia.store.es.default.auth.params.pwd=ductile
```

Comment:
it still a draft PR, but we must keep this commented, default properties are merged with provided ones.